donation-tracker/src/components/layout.tsx

```diff
@@ -87,6 +87,21 @@ const LayoutModule = (props: any) => {
     window.open(itemLink(), '_blank');
   };
 
+  // ToDo: this two functions are is a quick hack for two languages to start with; needs to become better over time
+  const handleLanguageSwitchClick = (event: React.MouseEvent<HTMLElement>) => {
+    event.preventDefault();
+    setAnchorElShare(null);
+    const languageIndex = languages.indexOf(language);
+    languageIndex === 0 ? changeLanguage(languages[1]) : changeLanguage(languages[0]);
```

Comment:
Hi, I'd use those `... ? ... : ...` thingy only, if you would also return something. Otherwise I'd use the usual `if ... then ... else`.

Here I'd call it like this
```
changeLanguage(languages.indexOf(language) === 0 ? languages[1] : languages[0]);
```

Reply:
Will remember for next time. As this is temporary code (until fully configurable i18n) I'll keep it here for now